drivers/clock_control/clock_control_rpi_pico.c

```diff
@@ -114,8 +127,12 @@
 #define AUXSTEM_clk_sys    CLOCKS_CLK_SYS_CTRL_AUXSRC_VALUE_
 #define AUXSTEM_clk_usb    CLOCKS_CLK_USB_CTRL_AUXSRC_VALUE_
 #define AUXSTEM_clk_adc    CLOCKS_CLK_ADC_CTRL_AUXSRC_VALUE_
-#define AUXSTEM_clk_rtc    CLOCKS_CLK_RTC_CTRL_AUXSRC_VALUE_
-#define AUXSTEM_clk_peri   CLOCKS_CLK_PERI_CTRL_AUXSRC_VALUE_
+#define AUXSTEM_clk_peri CLOCKS_CLK_PERI_CTRL_AUXSRC_VALUE_
+#if defined(CONFIG_SOC_SERIES_RP2040)
```

Comment:
ditto.

Reply:
As above. They're _only_ needed on a RP2040, which is indicated by the `#if #endif`